pkg/client/client.go

```diff
@@ -189,7 +191,7 @@ func (s *serviceImpl) HostService(host string) HostOrchestratorService {
 	hs := &HostOrchestratorServiceImpl{
 		HTTPHelper: s.httpHelper,
 		// Make the cloud orchestrator inject the credentials instead
-		BuildAPICredentialsHeader: headerNameCOInjectBuildAPICreds,
+		BuildAPICredentialsHeader: s.CredHeaderName,
```

Comment:
default to `X-Cutf-Host-Orchestrator-BuildAPI-Creds` here, and modify it down below with `headerNameCOInjectBuildAPICreds` if s.InjectBuildAPICreds is true.